service-base/src/main/java/org/eclipse/hono/service/management/Util.java

```diff
@@ -121,7 +123,7 @@ public static final Span newChildSpan(final String operationName, final SpanCont
      *
      * @param ctx The routing context of the request.
      * @param result The generic result of the operation.
-     * @param customHandler An (optional) handler for post processing the HTTP response, e.g. to set any additional HTTP
+     * @param customHandler An (optional) handler for post processing successful HTTP response, e.g. to set any additional HTTP
```

Comment:
can you please review the method's description in the JavaDoc? I don't think that it reflects the behavior of this method. It refers to a *returned handler* but the method returns `void`. It also doesn't mention that the body is only written and the handler is only invoked if the status code is 201 Created. IMHO the status code to check for should also be passed in as a parameter in order to make this method more useful. WDYT?

Reply:
Updated comments a bit. I wouldn't change the signature unless we actually have a case where we would reuse it.